lib/l10n/app_ru.arb

```diff
@@ -69,5 +68,10 @@
   },
   "dashScreenNavBarItemLabel": "Dash screen",
   "infoScreenNavBarItemLabel": "Info screen",
-  "debugScreenNavBarItemLabel": "Debug screen"
-}
+  "debugScreenNavBarItemLabel": "Debug screen",
+  "@segmentFeatureExample": {
+    "---------": "---------",
+    "description": "Example of feature"
+  },
+  "failedLoadIpMessage": "Failed to load IP address"
```

Comment:
The name of the variable must begin with the name of the segment. Here is the name pattern from the documentation: <segmentName><string procedure>